stdlib/3.9/zoneinfo/__init__.pyi

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+import os
+import typing
+from datetime import datetime, tzinfo
+from typing import IO, Optional, Sequence, Type, Union
+
+class ZoneInfo(tzinfo):
+    _T = typing.TypeVar("_T", bound="ZoneInfo")
```

Comment:
Typevars in typeshed are defined at top-level. This is possibly also why pytype chokes.